addon/usables/use-machine.js

```diff
@@ -93,17 +112,31 @@ export default function useMachine(machine, interpreterOptions = {}) {
 
   configurableMachineDefinition.machine = machine;
   configurableMachineDefinition.interpreterOptions = interpreterOptions;
+  configurableMachineDefinition.args = {
+    machine,
+    interpreterOptions,
+  };
+
+  configurableMachineDefinition.update = function (fn) {
```

Comment:
Maybe `onUpdate`? `.update()` doesn't seem like a function which takes a callback but one which immediately updates...

Naming is hard 😔

Reply:
the [use/resource-rfc](https://github.com/emberjs/rfcs/blob/use-and-resources/text/0567-use-and-resources.md#resource) suggests an `update`-method that's why I called this `update` instead of `onUpdate`

Reply:
Ah, didn't know. Thanks for the heads up 👍